upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -139,6 +140,8 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		if creds, err := spotinst.LoadCredentials(); err == nil {
 			dest["SpotinstToken"] = func() string { return creds.Token }
 			dest["SpotinstAccount"] = func() string { return creds.Account }
```

Comment:
I don't think these are used anywhere anymore.
Also, not quite sure how `SPOTINST_ACCOUNT_LEGACY` and `SPOTINST_TOKEN_LEGACY` are populated.

Reply:
Both are used [in the older manifest](https://github.com/kubernetes/kops/blob/15cc0fefae658bffa0dca7216d054918d70ebb8e/upup/models/cloudup/resources/addons/spotinst-kubernetes-cluster-controller.addons.k8s.io/v1.9.0.yaml.template#L10-L11) that points to an older version of the controller. I've kept the legacy variables just to maintain backward compatibility with the `ConfigMap`.

Reply:
Got it. Thanks!